README.md

```diff
@@ -169,6 +169,15 @@ will be publicly available after a few minutes on quay.io
 docker pull quay.io/opentargets/webapp:<yourbranchname>
 ```
 
+### Configuring analytics
```

Comment:
Documentation is good, but its not docker specific. Can this be extended and rearranged within the README a bit?

Reply:
Do you mean it should be `##` instead of `###`? It is indeed not meant to be docker specific.